### PR TITLE
Various Updates to User Documentation (LibreOffice Integration)

### PR DESCRIPTION
**Pull request by teertinker**
- Add External Tools.
- Add images
- A

### CODE REVIEW

File: en/advanced/resources.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ JabRef allows you to create custom export filters. This functionality and the in
 
 They are collected at [https://layouts.jabref.org](https://layouts.jabref.org).
 
+Styles files are available at [https://layouts.jabref.org](https://layouts.jabref.org) and for LibreOffice at [https://jstyles.jabref.org](https://jstyles.jabref.org).
```
Comment (koppor) [inline en/advanced/resources.md:33]: Then, the last paragraph can be deleted?

File: en/advanced/resources.md [hunk 2]
```diff
@@ -38,14 +40,14 @@ Homepage: [https://github.com/JabRef/EndNode-JabRef-filters](https://github.com/
...
-Using this tool you can easily create a custom export filter for JabRef to build you own bibliography style. Styles files are available at [https://layouts.jabref.org](https://layouts.jabref.org) and for LibreOffice at [https://jstyles.jabref.org](https://jstyles.jabref.org). The tool itself supports:
+Using this tool you can easily create a custom export filter for JabRef to build you own bibliography style. The tool itself supports:
 
 * HTML Export Filter
 * RTF Export Filter
 * OpenOffice/ LibreOffice Style File
 * Saving the filter for later refinements
 
-[Download the Export-Filter Editor](https://sourceforge.net/projects/efe/?source=dlp)
+[Download the Export-Filter Editor](https://github.com/teertinker/Export-Filter-Editor)
```
Comment (koppor) [inline en/advanced/resources.md:50]: Welcome to GitHub :)

**Review by koppor — COMMENTED** [review]
Just a small nitpick comment - otherwise: LGTM.

File: en/advanced/resources.md (hunk 1 above)
Comment (teertinker) [reply en/advanced/resources.md]: Almost. I did a small modification to keep the reference to jstyle files.

**Review by teertinker — COMMENTED** [review]
Thanks for the feedback. I added a small adjustment

**Review by Siedlerchr — CHANGES_REQUESTED** [review]
Thanks for the work! Can you please fix the markdownlint errors?

**Pull request merged**